cygnus-twitter/src/main/java/com/telefonica/iot/cygnus/sinks/TwitterSink.java

```diff
@@ -0,0 +1,482 @@
+/**
```

Comment:
This class only will make sense if you plan to create, in the future, other Twitter-2-something classes. Is that the case?